### ISSUE TITLE
Error report for 3.19 content.ggpk tecent 

### ISSUE THREAD

**Issue by Jock-SSH** [aianlinb/LibGGPK2#38]
Game version: 3.1901.2.1 Tencent version
Software version: VisualGGPK 0.15.1

Specific problems:
1)ROOT\Bundles2\Data\Simplified Chinese\ActiveSkills.dat
2)ROOT\Bundles2\Data\Simplified Chinese\ActiveSkills.dat64

After the above two files are exported in.csv format in VisualGGPK2, modify the.csv file and import it. Some active skills in the game cannot be used, and "this skill cannot be used" is displayed in the game.

Note: The text content of the.csv file is modified based on the patch file published by amsco2, and the replacement process is only for content other than numbers (such as the term "description").

Try Questions:
1) The skills can be used directly after the original content.ggpk is replaced, so the judgment is a problem when the file is exported and exported;
2) Try to export the.csv file and direct it back to the original path (without modifying any content). When you open the game, you find that the skill cannot be used;
3) Using the existing patch (Data.9.bundle) for direct replacement (using VisualGGPK) will not cause this problem.

In addition, after importing and replacing the.csv file through testing, there are still problems with CraftingBenchOptions and map launcher:
1) Process table: After opening the process table and putting in the equipment, the process affixes will display "equipment type error";
2) Map launcher: Originally, the first item was selected automatically (it seems to improve item rarity, I don't know), but now you need to select it manually.
Since the above two problems do not affect the use, the craft table can select the process affix according to the actual situation of the equipment, which can be marked with affix. The map launcher can also be used normally, so it does not affect the operation of the game intuitively. However, these two problems have not been further tested.

 Tip: Translator is used for translation ^_^.

**Comment by Jock-SSH** [comment]
Additional description: 
I ran some more tests on the above questions: 
Test Method: 
Operation 1) Export the dat64 file directly to the csv file, and import the dat64 file directly without modifying the csv file 
Note: 
The first exported file is: 
"BaseItemTypes.csv" 
BaseItemTypes.dat64(original file) 
The second exported file is as follows: 
"BaseItemTypes - 1.csv" 
"BaseItemTypes.dat64-1" (forget why I named it that way.^_^) 
For operation 1: 
After tests found that CSV and dat64 file file size are part of the increase 
(see P1: [https://s1.ax1x.com/2022/11/10/zpld6P.jpg] (url)). 
![2](https://user-images.githubusercontent.com/57992931/200986305-d99b2c91-054e-4c4a-990f-6519d66f3c76.jpg)

Operation 2) Export the imported dat64 file to the csv file for comparison. 
For Operation 2: 
Added some content in the CSV file, see the P2: [https://s1.ax1x.com/2022/11/10/zplwOf.jpg] (url).
![1](https://user-images.githubusercontent.com/57992931/200986282-94e29737-a47c-4575-9353-2cd2d2516085.jpg)
